submissions/ses3332002/friends-app/app.js

```diff
@@ -2,27 +2,33 @@
 const hamburgerMenu = document.querySelector(".hamburger_menu");
 const mainArea = document.querySelector("main");
 const optionForm = document.forms.sorting_n_filtering;
+const asIsOption = document.querySelector("#as_is");
+const allOption = document.querySelector("#all");
+const ageFromOption = document.querySelector("#age_from");
+const ageToOption = document.querySelector("#age_to");
+const searchNameOption = document.querySelector(".filter_name");
+const requestString = "https://randomuser.me/api/?nat=gb,us,fr&results=60&inc=gender,name,location,email,picture,dob";
 let receivedData = [];
 let dataToRender = [];
 let filterCriterionSex = () => true;
 let filterCriterionAge = () => true;
 let filterCriterionName = () => true;
-let sortCriterion = () => true;
-as_is.checked = true;
-all.checked = true;
-age_from.value = "";
-age_to.value = "";
-search_name.value = "";
+let sortCriterion = () => false;
+asIsOption.checked = "true";
+allOption.checked = "true";
+ageFromOption.value = "";
+ageToOption.value = "";
+searchNameOption.value = "";
```

Comment:
Do you really need this assignments? 
input text/number value is empty string by default
for input type radio you can set html attribute `cheched`

Reply:
I code on Linux and use Firefox for developing and testing. There is a bug, that Firefox does not clear form inputs on page reload, so I need to clear them this way.
I will use autocomplete="off" on a form to solve this issue.

Reply:
I remade it.